scripts/jenkins/jobs-ibs/templates/cloud-mkcloud-job-magnum-template.yaml

```diff
@@ -22,6 +22,6 @@
             nodenumbercompute=2
             want_magnum=1
             want_ceph=0
-            ostestroptions=" --regex '^magnum.tests.functional.api'"
+            ostestroptions= --regex '^magnum.tests.functional.api'
```

Comment:
@aplanas See https://ci.suse.de/job/openstack-mkcloud/32077/console for a real life example